scripts/release/versioning.mjs

```diff
@@ -16,7 +16,10 @@ const PLACEHOLDER_VERSION = 'PUBLISH_VERSION';
 const FILE_OPTIONS = { encoding: 'utf8' };
 
 const handler = async () => {
-  const packageJson = require(path.resolve(PACKAGE_ROOT, 'package.json'));
+  const packageJson = JSON.parse(
```

Comment:
Let's refactor this JSON paraser as a helper.

Reply:
Refactored